test/e2e/v3/e2e_suite.go

```diff
@@ -31,6 +31,9 @@ import (
 	"sigs.k8s.io/kubebuilder/test/e2e/utils"
 )
 
+const certmanagerVersion = "v1.0.1"
+const certmanagerURL = "https://github.com/jetstack/cert-manager/releases/download/%s/cert-manager-legacy.yaml"
```

Comment:
@MartinForReal you'll be able to customize which set of manifests to use (current vs legacy) with [kubectl version](https://github.com/kubernetes-sigs/kubebuilder/pull/1644/files#diff-5f5cf1a838c4d92e543721d2697ea0e2b0cf57b9f9a3c72bc9fa51a030bc56b8R80) once #1644 is merged.